File: support-utils/tests/src/com/hendraanggrian/support/utils/test/ViewGroupsActivity.java
package com.hendraanggrian.support.utils.test;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.ViewGroup;
import android.widget.ProgressBar;

/**
 * @author Hendra Anggrian (hendraanggrian@gmail.com)
 */
public class ViewGroupsActivity extends AppCompatActivity {

    public ProgressBar progressBar;
    public ViewGroup container;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_viewgroups);
        setSupportActionBar((Toolbar) findViewById(R.id.toolbar));
        progressBar = (ProgressBar) findViewById(R.id.progressBar);
        container = (ViewGroup) findViewById(R.id.container);
    }
}

File: tests/src/com/hendraanggrian/support/commonsutils/test/ViewGroupsActivity.kt
package com.hendraanggrian.support.commonsutils.test

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.Toolbar
import android.view.ViewGroup
import android.widget.ProgressBar

/**
 * @author Hendra Anggrian (hendraanggrian@gmail.com)
 */
class ViewGroupsActivity : AppCompatActivity() {

    var progressBar: ProgressBar? = null
    var container: ViewGroup? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_viewgroups)
        setSupportActionBar(findViewById(R.id.toolbar) as Toolbar)
        progressBar = findViewById(R.id.progressBar) as ProgressBar
        container = findViewById(R.id.container) as ViewGroup
    }
}